requirements.txt

```diff
@@ -0,0 +1,3 @@
+aiohttp==3.5.4
+Click==7.0
+Jinja2==2.10.1
```

Comment:
This file is also unneeded, the requirements are in poetry

https://github.com/saltstack/saltdocker/blob/master/pyproject.toml#L7-L11